requests-for-comments/2023-10-27-base3000-launch.md

```diff
@@ -0,0 +1,70 @@
+# Base 3000 & Notebooks launch
+
+We want to launch the ‘first version’ of PostHog 3000 after the next sprint, which means the week starting November 13th (+ 1 week if needed). We have decided to **include Notebooks** in this launch, so that we can have a more impactful launch that combines functionality and design changes.
+
+## Scope for launch
+
+- Dark Mode
+- Base 3000 [Check this mega issue for details](https://github.com/PostHog/posthog/issues/18089)
+- Notebooks side panel
+    
+<img width="674" alt="Screenshot 2023-10-27 at 10 38 50" src="https://github.com/PostHog/meta/assets/14750837/abb9e559-91ea-408b-a8fd-5105f3c4c626">
+
+## Out of scope for launch (amongst other things)
+
+- Sidebar (Index view)
+- Canvas-powered dashboards
+- Query builder
+- [Check this mega issue for details + the full list](https://github.com/PostHog/posthog/issues/18089)
+
+
+## Rollout plan
+- [ ] Base 3000 ready to release (10th Nov)
+- [ ] Internal and a/b testing starts (13th Nov)
+- [ ] Docs revamp and updates ready (24th Nov)
+- [ ] Product pages for 3000 / Notebooks (24th Nov)
+- [ ] Release [PostHog 3000 (Base 3000 scope)](https://app.posthog.com/feature_flags/8472) (27th Nov)
+- [ ] Release [Notebooks](https://app.posthog.com/feature_flags/8734) (4th Dec) (or together with Base 3000 if we feel docs are easier)
+
+
+## Pre-launch evaluation
+
+We decided that even the "base" version has enough major changes that we want to test that it both doesn't impact our key metrics negatively as well as (if possible) evaluate if it is overall an improvement, especially with new users.
+This might be hard to do or to get clear signals for whether it is an improvement (especially as it is only a step along the way to the end goal UI/UX).
+
+- [ ] **Signup/onboarding Experiment**: _just_ for the onboarding we could experiment with the styles separate from the rest of the app
+  - [ ] **Metrics**: Same as usual onboarding metrics?
+  - [ ] Important that we keep the style consistent from signup through to app usage as otherwise it will be a janky experience
+  - [ ] We mostly care about **negatively** impacting the flow
+- [ ] **General Experiment**: in combination with internal testing we could start to roll out the product to new and existing users as an experiment
+  - [ ] Important that we keep the style consistent from signup through to app usage as otherwise it will be a janky experience
+  - [ ] We originally opted against being able to go back to the old UI - do we want to offer this?
```

Comment:
Potentially for the period where we are running the A/B tests *before* the official launch, we can have a banner in the app *this is a design preview, you can revert to the old UI here*, so it's not such a novel experience without context, and people are less confused as to why the docs screenshots look different / they can help themselves

Reply:
I'm somewhat iffy on letting people opt out from an experimental perspective. I think it has the potential to muddy results. If someone is the kind of person who just hates abrupt change (most humans tbh!) they will opt out. But that doesn't mean it's a bad design. If they were forced to stick with it we actually get to see if they are able to get their work done even with the new design.

Similarly, people who like the change are more likely to be the early-adopter type and more likely to just explore around until they figure out how to do what they need to do. So we're getting too-positive signals from these people and too-negative signals from the former. 

I _think_ that if we can get confident about new users using A/B tests, we'll have more confidence about existing users. When we do A/B test existing users (if we want to, which is prob a good idea because they are more likely to be heavy users) we could maybe keep it in the feature preview and if they email support tell them how to disable?